MREUnityRuntime/MREUnityRuntimeLib/Constants.cs

```diff
@@ -23,15 +23,26 @@ static Constants()
             SerializerSettings.Converters.Add(new PayloadConverter());
         }
 
+        /*
+         * Headers passed in MRE connect requests.
+         */
         internal const string SessionHeader = "x-ms-mixed-reality-extension-sessionid";
         internal const string PlatformHeader = "x-ms-mixed-reality-extension-platformid";
-        internal const string ProtocolVersionHeader = "x-ms-mixed-reality-extension-protocol-version";
+        internal const string CurrentClientVersionHeader = "x-ms-mixed-reality-extension-client-version";
+        internal const string MinimumSupportedSDKVersionHeader = "x-ms-mixed-reality-extension-min-sdk-version";
+
+        /*
+         * Current Client Version
+         * This matches major.minor from the package version number, and is updated as part of the manual
+         * SDK release procedures.
+         */
+        internal const string CurrentClientVersion = "0.5";
 
         /*
-         * PROTOCOL VERSION
-         * This is the version of the wire protocol this host library speaks. This value is sent to the MRE app when
-         * connecting. If client's protocol version is unsupported, the connection will be rejected.
+         * Minimum Supported SDK version
+         * The oldest SDK version that runs. Since compatibility with older MREs is essential, changing
+         * this *is* a big deal, and requires discussion and signoff from the dev team.
          */
-        internal const int ProtocolVersion = 1;
+        internal const string MinimumSupportedSDKVersion = "0.5";
```

Comment:
>"0.5" [](start = 59, length = 5)

Won't this be 0.6 or later? The minimum supported version has to be the first SDK version that's actually checking these headers, right?